documentation/config_logging_advanced.markdown

```diff
@@ -69,7 +69,6 @@ To add JSON logging for HTTP requests:
 1.  Add the following Logback appender section to the `request-logging.xml` file:
 
     ``` xml
-    {% raw %}
```

Comment:
I believe that these are required for the tool that we use to convert our docs from the source code repo to the final HTML - @gguillotte can you confirm?